README.md

```diff
@@ -9,12 +9,12 @@ The goal of the Vulkan-Hpp is to provide header only C++ bindings for the Vulkan
 
 ## Getting Started
 
-Vulkan-Hpp is part of the LunarG Vulkan SDK since version 1.0.24. Just `#include <vulkan/vulkan.hpp>` and you're ready to use the C++ bindings. If you're using a Vulkan version not yet supported by the Vulkan SDK you can find the latest version of the header [here](https://github.com/KhronosGroup/Vulkan-Hpp/blob/master/vulkan/vulkan.hpp).
+Vulkan-Hpp is part of the [LunarG Vulkan SDK](https://www.lunarg.com/vulkan-sdk/) since version 1.0.24. Just `#include <vulkan/vulkan.hpp>` and you're ready to use the C++ bindings. If you're using a Vulkan version not yet supported by the Vulkan SDK, you can find the latest version of the header [here](vulkan/vulkan.hpp).

```

Comment:
I presume you meant https://github.com/KhronosGroup/Vulkan-Hpp/tree/main/vulkan? Since the link provided just links back to the same page (assuming the user is reading the README in the repo's root).